crates/common/src/traits.rs

```diff
@@ -181,6 +190,12 @@ impl TestFunctionKind {
         matches!(self, Self::UnitTest { .. })
     }
 
+    /// Returns `true` if this function is a `beforeTestSelectors` function.
+    #[inline]
+    pub const fn is_before_test(&self) -> bool {
```

Comment:
this is not a test kind

Reply:
changed with https://github.com/foundry-rs/foundry/pull/8497/commits/5213de6e0fd29d14fc2521d07b7d06c8382f35f0